prow/cmd/branchprotector/protect.go

```diff
@@ -34,10 +34,11 @@ import (
 )
 
 type options struct {
-	config    string
-	jobConfig string
-	confirm   bool
-	github    flagutil.GitHubOptions
+	config          string
+	jobConfig       string
+	confirm         bool
+	github          flagutil.GitHubOptions
+	printLineNumber bool
```

Comment:
If we're to add this to everything, can you add it to the `flagutils` package and DRY out the `logrus` setting?

Reply:
@stevekuznetsov Good find! I'll do it later today.

Reply:
Any ETA on when you'll be able to get back to this? Cool change!